docs/sdk/client-side-sdks/ios/ios-gettingstarted.md

```diff
@@ -69,22 +69,22 @@ For Objective-C we use a standard callback pattern to initialize the DevCycle SD
 }
 ```
 
-## DVC Client Builder
+## DevCycleClientBuilder
 
-The DVCClient can be built using the following methods:
+The DevCycleClient can be built using the following methods:
 
-[DVC ClientBuilder class](https://github.com/DevCycleHQ/ios-client-sdk/blob/4c69260ce89fd7c38245f48b99aa973e08ba05ca/DevCycle/DVCClient.swift#L371)
+[DevCycleClientBuilder class](https://github.com/DevCycleHQ/ios-client-sdk/blob/4c69260ce89fd7c38245f48b99aa973e08ba05ca/DevCycle/DevCycleClient.swift#L371)
```

Comment:
should reference the main instead of this old commit

https://github.com/DevCycleHQ/ios-client-sdk/blob/main/DevCycle/DevCycleClient.swift